01-intro.Rmd

```diff
@@ -314,3 +314,75 @@ check JSON input; unknown input/output: greetings.username
+
+
+### Fred Hutch Custom Runtime Variables
+For the `gizmo` cluster, the following custom runtime variables are available (below we show each variable with its current default value). You can change these variables in the `runtime` block for individual tasks in a WDL file. These variables are not variables that will be understood by Cromwell or other WDL engines when the workflow is not being run on the Fred Hutch cluster!  
+
+>Note: when values are specified in the runtime blocks of individual tasks in a workflow, those values will override these defaults for that task only!!
+
+
+- `walltime: "18:00:00"`
+  - A string ("HH:MM:SS") that specifies how much time you want to request for the task. Can also specify >1 day, e.g. "1-12:00:00" is 1 day+12 hours.
+- `partition:  "campus-new"`
+  - Which cluster partition to use. The default is `campus-new`: other options currently include `restart` or `short` but check [SciWiki](https://sciwiki.fredhutch.org/scicomputing/) for updated information. 
+- `modules: ""`
+  - A space-separated list of the environment modules you'd like to load (in that order) prior to running the task.  See below for more about software modules.
+- `dockerSL: `
+  - This is a custom configuration for the Hutch that allows users to use docker and softlinks only to specific locations in Scratch.  It is helpful when working with very large files. An example of the value for this variable is: `"ubuntu:latest"`. Just like the `docker:` runtime variable, only specify this if you want the task to run in a container (otherwise the default will be a non-containerized job). 
+- `account: `
+  - This allows users who run jobs for multiple PI accounts to specify which account to use for each task, to manage cluster allocations.  An example of the value for this variable is `"paguirigan_a"`, following the  pilastname_f pattern. 
+
+## Managing Software Environments
+
+
+### Modules
+At Fred Hutch we have huge array of pre-curated software modules installed on our SLURM cluster which you can [read about in SciWiki](https://sciwiki.fredhutch.org/scicomputing/compute_scientificSoftware/).   The custom configuration of our Cromwell server allows users to specify one or more modules to use for individual tasks in a workflow.  The desired module(s) can be requested for a task in the `runtime` block of your calls like this:
+
+```
+runtime {
+  modules: "GATK/4.2.6.1-GCCcore-11.2.0 SAMtools/1.16.1-GCC-11.2.0"
+}
+```
+
+In this example, we specify two modules, separated by a space (with quotes surrounding them). The GATK module will be loaded first, followed by the SAMtools module.   In this example you'll note the "toolchain" used to build each modules is the same ("GCC-11.2.0").  When you load >1 module for a single task it is important to ensure that they are compatible with each other.  Choose versions built with the same toolchain if you can.  
+
```

Comment:
I feel like these are the most relevant information, as the other un-highlighted part of the commit has been covered by other parts in the guide already. I would put this in the backend appendix that Ash is writing and have a toggle to hide it as its own section.

Reply:
okay. also i did just want to add this part alone to the intro not to main! cause main has added parts!